env.js

```diff
@@ -71,12 +71,60 @@ module.exports = (function () {
   };
 
   /**
-   * A JSON object that describes where to store intermediate files (data files to be processed)
-   * Known types are 'local' and 'sandcastle'.
+   * Credentials for AWS.  Only required if env.storage.default is 'aws/s3'.
    *
-   * 'local' is the default and just stores files locally
+   * The AWS_CREDENTIALS value should be a JSON string hash containing the
+   * properties 'accessKeyId' and 'secretAccessKey'.
+   *
+   * An example of a complete AWS_CREDENTIALS value:
+   *
+   * '{"accessKeyId": "<your-access-key-id>", "secretAccessKey": "<your-secret-access-key>"}'
+   */
+  env.awsCredentials = JSON.parse(config.fromEnvironment('AWS_CREDENTIALS', '{}'));
+
+  /**
+   * A JSON object that describes where to store intermediate files (data files
```

Comment:
Sure, I could, but there is a lot of other existing configuration here that isn't in a readme either.  Besides, a lot of this may (have to) change for a new infrastructure provider.

Thoughts?